HelpSource/Reference/AudioDeviceSelection.schelp

```diff
@@ -0,0 +1,172 @@
+
+note::
+Device selection won't take effect until the server is (re)booted.
+::
+
+section:: macOS
+
+If you want to explicitly tell the server to use the internal soundcard, you need to specify both input and output device. The following example comes from a MacBook Pro:
+code::
+Server.default.options.inDevice_("Built-in Microph");
+Server.default.options.outDevice_("Built-in Output");
+::
+
+In case of a dedicated audio interfaces, you might need to specify a single device, e.g.
+code::
+Server.default.options.device_("MOTU 828");
+::
+
+On macOS you can programmatically obtain list of available audio devices without booting the server:
+code::
+ServerOptions.devices; //all devices
+ServerOptions.inDevices; //input devices
+ServerOptions.outDevices; //output devices
+::
+
+subsection:: Sample rate mismatch
+
+One of the reasons for the server's failure to boot is the mismatch between input and output devices' sample rate. This is a possibility when using a different device for input and output (which is the case when using the internal soundcard). If the server fails to boot due to sample rate mismatch, an error will be posted accordingly.
+
+You should set both input and output devices' sample rate to the same value. You can do this in Audio MIDI Setup:
+code::
+"open -a 'Audio MIDI Setup'".unixCmd; // execute this to launch it
+::
+
+subsection:: Aggregate device
+
+Sometimes you might want to use multiple devices for input or output. In such case, macOS provides a functionality to combine multiple physical devices into one called an teletype::Aggregate Device::. To create one, you have to open the Audio MIDI Setup application (in /Applications/Utilities). You should do this from an user account with administrator privileges.
+code::
+"open -a 'Audio MIDI Setup'".unixCmd; // execute this to launch it
+::
+LIST::
+##In the teletype::Audio Devices:: window click on the Plus button on the bottom left and choose teletype::Create Aggregate Device::.
+##You can change its name to something specific if desired, for example "InternalAndMOTU".
+##Check the Use column on the right for the devices you want to combine.
+::
+Now you need to tell SuperCollider to use your new aggregate device.
+code::
+Server.default.options.device = "Aggregate Device"; //or the name you have specified in the previous step
+::
+
+After (re)booting the server (code::Server.default.reboot::) you should see in the post window that it now uses the Aggregate Device instead of system defaults:
+teletype::
+[...]
+"Aggregate Device" Input Device
+[...]
+"Aggregate Device" Output Device
+::
+
+Note that when you specify a sound device through SuperCollider's code::ServerOptions::, there is no need to use the aggregate device as the system's default device.
+
+
+section:: Linux
+
+By default SuperCollider on Linux uses JACK and the audio device selection is managed by the JACK server.
```

Comment:
Suggest to add `teletype::ServerOptions:: cannot override JACK's selection of audio hardware.`